commands_to_load/Mod.py

```diff
@@ -79,5 +79,390 @@ def __init__(self, bot):
+        await user.add_roles(MUTED_ROLE)
+        logger.info('[Mod mute()] adding muted role to user')
+
+        # Tell them in dm
+        logger.info('[Mod mute()] informing {} that they are muted in dm'.format(user))
+        eObj = em(title='Moderator action was taken against you', colour=0xff0000, description='You\'ve been muted in the CSSS server. Message a minion to learn why and how to be unmuted', footer='Moderator action')
+        await user.send(embed=eObj)
+        #TODO add something ^ to react to msg council for reconsideration slash reach out to u
+
+        # Tell council of action 
+        logger.info('[Mod mute()] getting council channel')
+        council = discord.utils.get(ctx.guild.channels, name='council')
+        logger.info('[Mod mute()] council channel found: {}'.format(council.id))
+
+        logger.info('[Mod mute()] informing council of {}\'s action to mute {}'.format(ctx.message.author, user))
+        eObj = em(description='{} muted {}'.format(ctx.message.author, user))
+        await council.send(embed=eObj)
+
+    @commands.command()
+    async def unmute(self, ctx):
+        # Unmutes user by removed the muted role from them
+
+        # Verify mention
+        logger.info('[Mod unmute()] checking for mention')
+        mentions = ctx.message.mentions
+        if len(mentions) != 1:
+            logger.info('[Mod unmute()] no mention found. Informing user')
+            eObj = em(description='You need to @ mention the user to mute', footer='Invalid arguments')
+            await ctx.send(embed=eObj, delete_after=5.0)
+            return
+        else: 
+            user = mentions[0]
+        logger.info('[Mod unmute()] user found through mention: {}'.format(user))
+
+        # Get muted role
+        MUTED_ROLE = discord.utils.get(ctx.guild.roles, name='Muted')
+        logger.info('[Mod unmute()] muted role found: {}'.format(MUTED_ROLE.id))
+
+        # Verify user has the muted role
+        logger.info('[Mod unmute()] verifying if {} is muted or not'.format(user))
+        if user not in MUTED_ROLE.members:
+            logger.info('[Mod unmute()] {} is not muted. Informing {} of this fact'.format(user, ctx.message.author))
+            eObj = em(description='{} is not muted so cannot unmute'.format(user), footer='Invalid arguments')
+            await ctx.send(embed=eObj)
+            return
+        logger.info('[Mod unmute()] {} verified to be muted'.format(user))
+
+        # Remove role from the user
+        await user.remove_roles(MUTED_ROLE)
+        logger.info('[Mod unmute()] removed muted role from {}'.format(user))
+
+        # Tell user of their new freedom and to not abuse it
+        logger.info('[Mod unmute()] letting {} know they\'ve been unmuted and to not do something stupid again')
+        eObj = em(description='You\'ve been unmuted. Don\'t do whatever you did to get muted in the first place again, or else next time you\'ll get more than a ban', footer='Moderator action')

```

Comment:
`or else next time you'll get more than a ban`: is `ban` meant here or `mute`?

Reply:
nice catch. wast supposed to say ban